packages/radix-ui-themes/src/components/data-list.props.ts

```diff
@@ -0,0 +1,83 @@
+import { PropDef, trimProp } from '../helpers';
+
+const orientationValues = ['horizontal', 'vertical'] as const;
+const gapValues = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'] as const;
+const alignValues = ['start', 'center', 'end', 'baseline'] as const;
+const sizes = ['1', '2', '3'] as const;
+
+export const dataListPropDefs = {
+  orientation: {
+    type: 'enum',
+    className: 'rt-r-o',
+    values: orientationValues,
+    default: 'horizontal',
+    responsive: true,
+  },
+  gap: { type: 'enum', className: 'rt-r-gap', values: gapValues, default: '4', responsive: true },
+  gapX: {
+    type: 'enum',
+    className: 'rt-r-cg',
+    values: gapValues,
+    default: undefined,
+    responsive: true,
+  },
+  gapY: {
+    type: 'enum',
+    className: 'rt-r-rg',
+    values: gapValues,
+    default: undefined,
+    responsive: true,
+  },
+  size: {
+    type: 'enum',
+    className: 'rt-r-size',
+    values: sizes,
+    default: '2',
+    responsive: true,
+  },
+  trim: {
+    ...trimProp,
+    className: 'rt-r-dl-lt', // Custom trim styles due to grid layout
```

Comment:
I used `dl` for data list, not sure exactly the best name to use here.

Reply:
@benoitgrelard I also wondered about this one, do you have any preference about how to name classes that have overlap with other utilities like this one?